templates/package-json.js

```diff
@@ -1,16 +1,67 @@
 module.exports = (answers) => {
-	return `{
-	"private": true,
-	"name": "${answers.name}",
-	"devDependencies": {
-		"eslint": "^7.1.0",
-		"eslint-config-origami-component": "^2.0.1",
-		"origami-ci-tools": "^2.0.0",
-		"remark": "^12.0.0",
-		"remark-lint": "^7.0.0",
-		"remark-preset-lint-origami-component": "^1.1.1",
-		"stylelint": "^13.5.0",
-		"stylelint-config-origami-component": "^1.0.2"
+
+	const packageJson = {
+		"name": `@financial-times/${answers.name}`,
+		"version": "0.0.0",
+		"description": answers.description,
+		"keywords": answers.keywords,
+		"homepage": `https://registry.origami.ft.com/components/${answers.name}`,
+		"bugs": {
+			"url": `https://github.com/Financial-Times/${answers.name}/issues`
+		},
+		"repository": {
+			"type": "git",
+			"url": `git+https://github.com/Financial-Times/${answers.name}.git`
+		},
+		"license": "MIT",
+		"author": "",
+		"type": "module",
+		"devDependencies": {
+			"remark": "^12.0.1",
+			"remark-lint": "^7.0.1",
+			"remark-preset-lint-origami-component": "^2.0.0-beta.1"
```

Comment:
```suggestion
			"remark-preset-lint-origami-component": "^2.0.0"
```

Reply:
fixed in [`1b0e344` (#238)](https://github.com/Financial-Times/create-origami-component/pull/238/commits/1b0e3449b950710fc2defeaab53eac4faeb36ea9)